packages/discovery/src/discovery/engine/DiscoveryEngine.ts

```diff
@@ -6,6 +6,8 @@ import { shouldSkip } from './shouldSkip'
 
 // Bump this value when the logic of discovery changes,
 // causing a difference in discovery output
+
+// Last change: removed `code` key from the output
```

Comment:
should this comment be there, the logic version has not been changed

Reply:
My bad, the version should be changed

Reply:
9246c51a

Reply:
9246c51a